### PR TITLE
update master into leaflet-1.X

### PR DESCRIPTION
**Pull request by nmccready**
Updates post fork

### CODE REVIEW

**Comment by nmccready** [comment]
I don't expect tests to pass but this PR is to acknowledge keeping master updates into leaflet-1.x

File: package.json [hunk 1]
```diff
@@ -71,6 +71,6 @@
   "dependencies": {
     "angular": "1.x",
     "angular-simple-logger": "~0.1.4",
-    "leaflet": "0.7.x"
+    "leaflet": "1.0.0-beta.1"
```
Comment (nmccready) [inline package.json]: Fix later but why is this not beta.2?

**Comment by nmccready** [comment]
@elesdoar anything missing?

File: package.json (hunk 1 above)
Comment (elesdoar) [inline package.json]: this should be beta 2, I'll check it

**Comment by elesdoar** [comment]
Some test fail for incompatibility with leaflet 1.x

**Comment by elesdoar** [comment]
I closing this PR for #172 PR, resolve all conflicts.

**Comment by nmccready** [comment]
Ahh this was targeted the wrong way my bad.. nice catch.

**Pull request closed without merging**